apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V36__preadj_api_source_column.sql

```diff
@@ -0,0 +1,5 @@
+/*
+ * Add column used to track version of server that produced each claim.
+ */
+alter table "pre_adj"."FissClaims" add "apiSource" varchar(64);
```

Comment:
can we safely make this smaller, maybe a varchar(16) or varchar(32)?

Reply:
16 won't work because of the length of the timestamp for S3.  I have reduced it to 24.  I think that should cover all the cases.